Docs/ProjectSpec.md

```diff
@@ -133,6 +134,27 @@ options:
   postGenCommand: pod install
 ```
 
+### GroupOrdering
+
+Describe an order of groups. Available parameters:
+
+- [ ] **pattern**: **String** - A group name pattern. Can be just a single string and also can be a regex pattern. Optional option, if you don't set it, it will pattern for the main group, i.e. the project.
+- [ ] **order**: **[String]** - An order of groups.
+- [ ] **fileSortPosition**: **String** - Where files are sorted in relation to other groups. Either:
+  - `top` (default) - at the top, before groups
+  - `bottom` - at the bottom, after other groups
+
```

Comment:
Is this required? Would someone ever want it to be different than `options.groupSortPosition`